contracts/upgradeable/mocks/ERC721ReceiverMockUpgradeable.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.18;
+
+import "@openzeppelin/contracts-upgradeable/token/ERC721/IERC721ReceiverUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+contract ERC721ReceiverMockUpgradeable is
+    Initializable,
+    IERC721ReceiverUpgradeable
+{
```

Comment:
I do not think we need this. Unless it is inherited by another mock we can test with regular ERC721.